charts/tidb-operator/templates/admission/admission-webhook-deployment.yaml

```diff
@@ -28,6 +28,10 @@ spec:
 {{ toYaml .Values.imagePullSecrets | indent 6 }}
     {{- end }}
       serviceAccountName: {{ .Values.admissionWebhook.serviceAccount }}
+    {{- with .Values.advancedStatefulset.securityContext }}
```

Comment:
This is for webhook.

Reply:
Similar comments for the other components.

Reply:
I'll fix